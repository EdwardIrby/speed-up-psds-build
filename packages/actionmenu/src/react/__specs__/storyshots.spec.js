import initStoryshots, {
  snapshotWithOptions
} from '@storybook/addon-storyshots'
import React from 'react'

function createNodeMock(element) {
  return { focus() {} }
}

initStoryshots({
  test: snapshotWithOptions({
    createNodeMock
  })
})
