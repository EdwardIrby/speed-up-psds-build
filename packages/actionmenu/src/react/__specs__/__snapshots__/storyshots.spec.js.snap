// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots dividers edge 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-1mfexnt"
    >
      One item
    </div>
    <div
      className="css-rficis"
    />
  </div>
</div>
`;

exports[`Storyshots dividers sandwich 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-1mfexnt"
    >
      One item
    </div>
    <div
      className="css-rficis"
    />
    <div
      className="css-1mfexnt"
    >
      Two item
    </div>
    <div
      className="css-1mfexnt"
    >
      Three item
    </div>
  </div>
</div>
`;

exports[`Storyshots menu items long text 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-1mfexnt"
    >
      One item that has text that goes on forever and onward into the universes yet to be
    </div>
  </div>
</div>
`;

exports[`Storyshots menu items long text with icon 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-1rwsoga"
    >
      <div
        className="css-18rfdc6"
      >
        <div
          className="css-otejxm"
        >
          <svg
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M13 13.855V23h-2v-9.145c-1.4457-.4302-2.5-1.7695-2.5-3.355C8.5 8.567 10.067 7 12 7s3.5 1.567 3.5 3.5c0 1.5855-1.0543 2.9248-2.5 3.355zM5.0412 1.3437L6.2514 2.936C3.9716 4.6715 2.5 7.414 2.5 10.5s1.4716 5.8286 3.7514 7.564l-1.2102 1.5924C2.2814 17.5557.5 14.236.5 10.5c0-3.736 1.7814-7.0557 4.5412-9.1564zm13.9176 0C21.7186 3.4443 23.5 6.764 23.5 10.5c0 3.736-1.7814 7.0557-4.5412 9.1564l-1.2102-1.5925C20.0284 16.3284 21.5 13.586 21.5 10.5s-1.4716-5.8286-3.7514-7.564l1.2102-1.5924zM7.4616 4.5287l1.2104 1.592C7.352 7.1255 6.5 8.7133 6.5 10.5c0 1.7867.852 3.3745 2.172 4.379l-1.2104 1.5925C5.6618 15.1015 4.5 12.9365 4.5 10.5c0-2.4365 1.1618-4.6016 2.9616-5.9715zm9.0768 0C18.3382 5.8982 19.5 8.0634 19.5 10.5s-1.1618 4.6016-2.9616 5.9715L15.328 14.879c1.32-1.0045 2.172-2.5923 2.172-4.379 0-1.7867-.852-3.3745-2.172-4.379l1.2104-1.5925zM12 12c.8284 0 1.5-.6716 1.5-1.5S12.8284 9 12 9s-1.5.6716-1.5 1.5.6716 1.5 1.5 1.5z"
            />
          </svg>
        </div>
      </div>
      One item that has text that goes on forever and onward into the universes yet to be
    </div>
  </div>
</div>
`;

exports[`Storyshots menu items multiple 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-1mfexnt"
    >
      One item
    </div>
    <div
      className="css-1mfexnt"
    >
      Two item
    </div>
    <div
      className="css-1mfexnt"
    >
      Three item
    </div>
  </div>
</div>
`;

exports[`Storyshots menu items one 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-1mfexnt"
    >
      One item
    </div>
  </div>
</div>
`;

exports[`Storyshots menu items with icons 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-1rwsoga"
    >
      <div
        className="css-18rfdc6"
      >
        <div
          className="css-otejxm"
        >
          <svg
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M13 13.855V23h-2v-9.145c-1.4457-.4302-2.5-1.7695-2.5-3.355C8.5 8.567 10.067 7 12 7s3.5 1.567 3.5 3.5c0 1.5855-1.0543 2.9248-2.5 3.355zM5.0412 1.3437L6.2514 2.936C3.9716 4.6715 2.5 7.414 2.5 10.5s1.4716 5.8286 3.7514 7.564l-1.2102 1.5924C2.2814 17.5557.5 14.236.5 10.5c0-3.736 1.7814-7.0557 4.5412-9.1564zm13.9176 0C21.7186 3.4443 23.5 6.764 23.5 10.5c0 3.736-1.7814 7.0557-4.5412 9.1564l-1.2102-1.5925C20.0284 16.3284 21.5 13.586 21.5 10.5s-1.4716-5.8286-3.7514-7.564l1.2102-1.5924zM7.4616 4.5287l1.2104 1.592C7.352 7.1255 6.5 8.7133 6.5 10.5c0 1.7867.852 3.3745 2.172 4.379l-1.2104 1.5925C5.6618 15.1015 4.5 12.9365 4.5 10.5c0-2.4365 1.1618-4.6016 2.9616-5.9715zm9.0768 0C18.3382 5.8982 19.5 8.0634 19.5 10.5s-1.1618 4.6016-2.9616 5.9715L15.328 14.879c1.32-1.0045 2.172-2.5923 2.172-4.379 0-1.7867-.852-3.3745-2.172-4.379l1.2104-1.5925zM12 12c.8284 0 1.5-.6716 1.5-1.5S12.8284 9 12 9s-1.5.6716-1.5 1.5.6716 1.5 1.5 1.5z"
            />
          </svg>
        </div>
      </div>
      One item
    </div>
    <div
      className="css-1rwsoga"
    >
      <div
        className="css-18rfdc6"
      >
        <div
          className="css-otejxm"
        >
          <svg
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4.4113 10l3.5858 3.5858V22H6l-.003-7.5858L3 11.4172V14H1V8h6v2H4.4113zM21 11.4142l-3 3V22h-2v-8.4142L19.5858 10H17V8h6v6h-2v-2.5858zm-8-5.5884V22h-2V5.831L9.1742 7.657 7.76 6.2425 12.0026 2l4.2427 4.2426L14.831 7.657 13 5.8257z"
            />
          </svg>
        </div>
      </div>
      Two item
    </div>
    <div
      className="css-1rwsoga"
    >
      <div
        className="css-18rfdc6"
      >
        <div
          className="css-otejxm"
        >
          <svg
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M22 11c0-.551-.448-1-1-1h-4c-.552 0-1 .449-1 1v9h-1v-4c0-.551-.448-1-1-1h-4c-.552 0-1 .449-1 1v4H8v-7c0-.551-.448-1-1-1H3c-.552 0-1 .449-1 1v7H0v2h24v-2h-2v-9zM4 20v-6h2v6H4zm7 0v-3h2v3h-2zm7 0v-8h2v8h-2zM5 9c-1.104 0-2-.896-2-2s.896-2 2-2c1.102 0 1.994.891 1.999 1.992l4.002 2.286C11.296 9.106 11.635 9 12 9c.178 0 .347.031.511.074l4.563-4.562C17.031 4.349 17 4.179 17 4c0-1.104.896-2 2-2s2 .896 2 2-.896 2-2 2c-.178 0-.348-.029-.512-.073l-4.562 4.562c.043.165.074.333.074.511 0 1.104-.896 2-2 2-1.102 0-1.994-.89-1.999-1.99L5.999 8.724C5.704 8.896 5.365 9 5 9z"
            />
          </svg>
        </div>
      </div>
      Three item
    </div>
  </div>
</div>
`;

exports[`Storyshots nested origin bottomLeft 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-wyi7wi"
    >
      <div
        className="css-kfg7gj"
      >
        One item
        <div
          className="css-c4zu2z"
        >
          <div
            className="css-wyi7wi"
          >
            <div
              className="css-kfg7gj"
            >
              One nested item
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots nested origin bottomRight 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-wyi7wi"
    >
      <div
        className="css-kfg7gj"
      >
        One item
        <div
          className="css-c4zu2z"
        >
          <div
            className="css-wyi7wi"
          >
            <div
              className="css-kfg7gj"
            >
              One nested item
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots nested origin topLeft 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-wyi7wi"
    >
      <div
        className="css-kfg7gj"
      >
        One item
      </div>
    </div>
    <div
      className="css-rficis"
    />
  </div>
</div>
`;

exports[`Storyshots nested origin topRight 1`] = `
<div
  style={
    Object {
      "background": "transparent",
      "backgroundPosition": "center",
      "backgroundSize": "cover",
      "bottom": 0,
      "left": 0,
      "overflow": "scroll",
      "position": "fixed",
      "right": 0,
      "top": 0,
      "transition": "background 0.25s ease-in-out",
    }
  }
>
  <div
    className="css-c4zu2z"
  >
    <div
      className="css-wyi7wi"
    >
      <div
        className="css-kfg7gj"
      >
        One item
        <div
          className="css-c4zu2z"
        >
          <div
            className="css-wyi7wi"
          >
            <div
              className="css-kfg7gj"
            >
              One nested item
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
